dev/InfoBar/InfoBar.xaml

```diff
@@ -0,0 +1,203 @@
+                                </VisualState>
+                                
+                                <VisualState x:Name="UserIconVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="UserIconBorder.Visibility" Value="Visible"/>
+                                        <Setter Target="StandardIcon.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                
+                                <VisualState x:Name="NoIconVisible"/>
+                            </VisualStateGroup>
+
+                            <VisualStateGroup>
+                                <VisualState x:Name="CloseButtonVisible"/>
+                                <VisualState x:Name="CloseButtonCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="CloseButton.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="InfoBarVisibility">
+                                <VisualState x:Name="InfoBarVisible"/>
+                                <VisualState x:Name="InfoBarCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="ContentRoot.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            
+                        </VisualStateManager.VisualStateGroups>
+
+                        <!-- Background is used here so that it overrides the severity status color if set. -->
+                        <Grid
+                            HorizontalAlignment="Stretch"
+                            MinHeight="{ThemeResource InfoBarMinHeight}"
+                            Background="{TemplateBinding Background}"
+                            Padding="{StaticResource InfoBarContentRootPadding}">
+
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto"/> <!-- Icon -->
+                                <ColumnDefinition Width="*"/>    <!-- Title, message, and action -->
+                                <ColumnDefinition Width="Auto"/> <!-- Close button -->
+                            </Grid.ColumnDefinitions>
+
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto"/>
+                                <RowDefinition Height="Auto"/>
+                            </Grid.RowDefinitions>
+
+                            <TextBlock x:Name="StandardIcon"
```

Comment:
>TextBlock [](start = 29, length = 9)

Why is this a textBlock instead of a FontIcon?

Reply:
No particular reason. I can change it. Why is FontIcon better?

Reply:
I don't think there is much difference, FontIcon was tailored for fonts - @MikeHillberg might know if there are any differences.

Reply:
SymbolIcon is easier to read and use, but doesn't include every symbol available in the font, so sometimes FontIcon is necessary.